src/cohort_manager.js

```diff
@@ -0,0 +1,117 @@
+const Cohorts = require('./cohorts.js')
+
+class CohortManager {
+  constructor() {
+    this.cohortList = []
+    this.studentId = 1
+    this.max_capacity = 5
+  }
+
+  createCohort(cohortName) {
+    const cohortToAdd = new Cohorts(cohortName)
+    this.cohortList.push(cohortToAdd)
+    return this.cohortList
+  }
+
```

Comment:
Good error checking by initializing a variable set to false, then handling that condition.

Generally your logic looks sound. All good !!